Tests/run_tests.py

```diff
@@ -70,6 +70,13 @@ def is_numpy():
         return False
 
 
+def is_outside_bio(name):
+    name = name.split(".")
+    if "Bio" in name[0]:
```

Comment:
I'd use ``name[0].startswith("Bio")`` or maybe even ``name[0] in ["Bio", "BioSQL"]`` to be explicit?